src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final double PI = 3.14159265359;
+    private double radius;
+
+    public Circle(double radius, ColorSupplier colorSuppliers) {
+        super(colorSuppliers);
+        this.radius = radius;
+    }
+
+    public Circle(double radius) {
+        super(defaultColor);
+        this.radius = radius;
+    }
+
+    public void draw() {
```

Comment:
Consider using of Override annotation here and in the following cases

Reply:
not fixed. Please add Override annotaion